### PR TITLE
adds error reporting quickstart

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: error_reporting/quickstart/test/quickstartTest.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        );
+        file_put_contents($file, $contents);
+
+        // Invoke quickstart.php
+        ob_start();
+        passthru(sprintf('php %s', $file));
+        $output = ob_get_clean();
+
+        // Make sure it looks correct
+        $this->assertEquals(
```
Comment (tmatsuo) [inline error_reporting/quickstart/test/quickstartTest.php]: This is testing whether or not it succesfully wrote the log to Cloud Logging API.
Is there a way to make sure it's correctly handled by the Error Reporting?

**Comment by waprin** [comment]
lgtm

File: error_reporting/quickstart/test/quickstartTest.php (hunk 1 above)
Comment (bshaffer) [inline error_reporting/quickstart/test/quickstartTest.php]: @tmatsuo I can probably make an API call, yeah

Comment (bshaffer) [inline error_reporting/quickstart/test/quickstartTest.php]: Done!

File: error_reporting/quickstart/test/quickstartTest.php [hunk 2]
```diff
@@ -41,5 +47,42 @@ public function testQuickstart()
             'Exception logged to Stack Driver Error Reporting' . PHP_EOL,
             $output
         );
+
+        // Ensure the log actually showed up
+        sleep(1);
```
Comment (tmatsuo) [inline error_reporting/quickstart/test/quickstartTest.php]: I think there is EventualConsistentTestRunner somewhere. Does it make more sense to use that?

Comment (bshaffer) [inline error_reporting/quickstart/test/quickstartTest.php]: Good call! Added.

**Pull request merged**